lib/msf/core/post_mixin.rb

```diff
@@ -38,8 +39,6 @@ def setup
       print_warning('SESSION may not be compatible with this module.')
     end
 
-    super
```

Comment:
Fixed in https://github.com/rapid7/metasploit-framework/commit/b2de5aba07c0b0e9379caa7373d717c93c666341. This was not dead code for local exploits.